### PR TITLE
UIP now runs as a service on linux

### LINKED ISSUE #97: Create UIP as a service
Keeping command prompt open all the time is always a problem for me if UIP could run as service in background like stay in system tray, that would be really awesome as a user for me.

### PR DESCRIPTION
**Pull request by DarkSouL11**
To run UIP as a service: UIP --service [option]
option = start, to start
           = stop, to stop
           = restart, to restart
Can also be combined with other available flags.
Also a change to be noted is, now press 's' + enter when running as a script to skip current wallpaper
Partially fixes issue 97
along with a few minor fixes.

Fixes #97

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,8 +16,10 @@ def get_contents(filename):
     return out
 
 #Make Home Directory
-if not os.path.exists(HOME_DIR):
-    os.makedirs(HOME_DIR)
+if os.path.exists(HOME_DIR):
+    shutil.rmtree(HOME_DIR)
```
Comment (nkprince007) [inline setup.py]: o_O
Are you aware of what this does? @DarkSouL11 

This removes all the directories in "~"

File: setup.py [hunk 2]
```diff
@@ -15,8 +16,10 @@ def get_contents(filename):
...
+
```
Comment (nkprince007) [inline setup.py]: why is this removal necessary?

Comment (DarkSouL11) [inline setup.py]: Thats useful for debugging purposes.When you install UIP for first time settings.json file will be saved in .uip folder but after when you change any option like timeout manually in code and reinstall it wasn't being overwritten don't know why that was happening.So, I am just deleting the settings of any previous installation if available.

Comment (nkprince007) [inline setup.py]: Will check it on OSX and comment back later..

Comment (DarkSouL11) [inline setup.py]: Yeah eager to know if it works :P

File: uiplib/daemon.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
```
Comment (abhsag24) [inline uiplib/daemon.py]: can we move this to an own file?

Comment (DarkSouL11) [inline uiplib/daemon.py]: what do you mean by an own file?

**Comment by abhsag24** [comment]
Like a new file that just holds the daemon class

**Comment by abhsag24** [comment]
Ohh its already there sorry in the ui it seemed like the same file

**Comment by DarkSouL11** [comment]
It is in a new file name  `daemon.py`

**Comment by DarkSouL11** [comment]
Yeah we can't differentiate between files on this UI :P

File: uiplib/daemon.py (hunk 3 above)
Comment (nkprince007) [inline uiplib/daemon.py]: I don't use `/usr/bin/python`. People like me, using `virtualenv`, can have issues with that.

**Comment by nkprince007** [comment]
@DarkSouL11 it works on macOS too.. just add `sys.platform.startswith('darwin')` to add support for macOS

**Comment by nkprince007** [comment]
Don't know why, but using proxy, does seem to have issues with downloading wallpapers.

**Comment by DarkSouL11** [comment]
Does it download without proxy?

**Comment by nkprince007** [comment]
Sometimes.. Not always..

**Comment by abhsag24** [comment]
closed by my PR, awesome work in anycase :D !
